dune/stuff/common/algorithm.hh

```diff
@@ -8,21 +8,23 @@
 
 #include <utility>
 
-namespace Dune {
-namespace Stuff {
-namespace Common {
+namespace Dune
+{
+namespace Stuff
+{
+namespace Common
+{
 
 //! same as std::copy_if, but with move
-template<class InputIt, class OutputIt, class UnaryPredicate>
-OutputIt move_if(InputIt first, InputIt last,
-                 OutputIt d_first, UnaryPredicate pred)
+template < class InputIt, class OutputIt, class UnaryPredicate >
```

Comment:
I really want to have the spaces inside `< ... >` so I am happy with that. I could not find way, however, to (leave the) remove(d) space _before_ the ´<`. I would have liked the above line to read
```
template< class InputIt, class OutputIt, class UnaryPredicate >
```
This is not a show stopper for me.